client/extensions/wp-super-cache/preload-tab.jsx

```diff
@@ -30,28 +30,37 @@ import { isPreloadingCache } from './state/cache/selectors';
  */
 const renderCachePreloadInterval = ( {
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/14518/commits/b4c366ddb946c929437faa030ff2e0739bb746a3.